### ISSUE TITLE
 Couldn't find 'checkpoint' file or checkpoints

### ISSUE THREAD

**Issue by avisekiit** [JiahuiYu/generative_inpainting#46]
Hello,
Thanks for sharing the codebase publicly. So, after I clone the repo, I downloaded the pretrained models for HQ CelebA and tried running the test.py code (Tensorflow version:1.6.0). I created a folder model_logs and placed all the files of the downloaded pre-trained model in that folder. However, I get the error that:
` Couldn't find 'checkpoint' file or checkpoints in given directory model_logs/' 

I run using the command:
python test.py --image examples/celeba/celebahr_patches_164036_input.png --mask examples/center_mask_256.png --output examples/output.png --checkpoint_dir model_logs/

And the contents of model_logs are:

checkpoint.txt  
snap-0.data-00000-of-00001  
snap-0.index  
snap-0.meta  
train_shuffled.flist  
val_shuffled.flist

Can you please let me know the correct way to just run your pretrained models.

Thanks,
Avisek

**Comment by JiahuiYu** [comment]
Download the model dirs and put it under model_logs/ (**rename checkpoint.txt to checkpoint because google drive automatically add ext after download**).

**Comment by horizonheart** [comment]
> checkpoint.txt

there is no checkpoint.txt
![image](https://user-images.githubusercontent.com/16372054/47473460-3cd90a00-d845-11e8-963e-d4421de7bfc9.png)

**Comment by JiahuiYu** [comment]
@horizonheart Are you serious? Our code is for image inpainting and models can be downloaded with links provided.
